src/types/Todo.ts

```diff
@@ -0,0 +1,6 @@
+export interface Todo {
+  id: number;
```

Comment:
Consider renaming `Todo` interface to `ITodo` to follow common naming conventions for interfaces. 🤔